challenge-01.html

```diff
@@ -140,30 +147,30 @@ <h3>Cameras that catch any shot.</h3>
   
         <ul>
           <li>
-            <!-- Needs an image (<img width="" height="" ...>) width 165 google-pixel-3a.png -->
+            <!-- Needs an image (Remember: <width="" height="" ...>) width of 165px google-pixel-3a.png -->
```

Comment:
This should be `<img width=“”>` there is not `<width>` tag!

Reply:
Thank you for catching that! I'll fix right away.